test/test.js

```diff
@@ -13,6 +13,7 @@ describe('DynamoDBStore', function () {
     describe('Instantiation', function () {
         it('should be able to be created', function () {
             var store = new DynamoDBStore({
+                ttl: false,
```

Comment:
The local dynamodb I am testing against does not appear to support the new ttl functionality.